domains/gary.json

```
@@ -6,6 +6,6 @@
     "email": "contact@garybot.dev"
   },
   "record": {
-    "URL": "https:/garylovesclimbingtreesexploringplacesandchasingbutterflies.garythecatisanadorablefelinealwaysseekingfunandadventure.site"
+    "cname": "garylovesclimbingtreesexploringplacesandchasingbutterflies.garythecatisanadorablefelinealwaysseekingfunandadventure.site"
```

`cname` has to be in all caps iirc
```suggestion
    "CNAME": "garylovesclimbingtreesexploringplacesandchasingbutterflies.garythecatisanadorablefelinealwaysseekingfunandadventure.site"
```